projects/js-packages/licensing/components/activation-screen/utils.js

```diff
@@ -63,13 +63,3 @@ export function getProductGroup( productId ) {
 		'default'
 	);
 }
-
-/**
- * Check if a plugin is active.
- *
- * @param {string} plugin -- The name of the plugin such as jetpack/jetpack.php, social/jetpack-social.php, backup/jetpack-backup.php.
- * @returns {boolean} -- True or False.
- */
-export function isPluginActive( plugin ) {
-	return window.myJetpackInitialState?.plugins?.[ plugin ]?.active;
-}
```

Comment:
If we get rid of this, maybe we could also stop fetching plugins in My Jetpack? That would be a nice performance improvement:
https://github.com/Automattic/jetpack/blob/60884bc6bd01b21b80431d6fc3c2f8b6a8124e2d/projects/packages/my-jetpack/src/class-initializer.php#L203
https://github.com/Automattic/jetpack/blob/cc0663db3d9d15cca0b7ae1746a1bff555bdd823/projects/packages/my-jetpack/global.d.ts#L30-L48

That's assuming `myJetpackInitialState.plugins` isn't used anywhere else.

Edit: I now see it's used in https://github.com/Automattic/jetpack/blob/cc0663db3d9d15cca0b7ae1746a1bff555bdd823/projects/packages/my-jetpack/_inc/state/selectors.js#L105-L118
That would imply a bit of refactoring in there as well.